edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.Http.Test/AuthenticationMiddlewareTest.cs

```diff
@@ -328,30 +304,6 @@ public async Task InvalidCredentialsRequestX509Test_AuthFailed()
             Assert.Equal("Unable to authenticate device with Id device_2/module_1", result.message);
         }
 
-        [Fact]
-        public async Task InvalidAuthenticateRequestX509Test_InvalidDeviceId()
-        {
```

Comment:
Varun: don't remove the tests, just disable for now until we have the authentication middleware doing the right thing with multiple service APIs.